### DISCUSSION TITLE
Standardize `IIF.Position{N}`

### DISCUSSION THREAD

**Discussion by dehann** [JuliaRobotics/Caesar.jl#807] [Ideas]
Why not consolidate:
- [ ] `IIF.ContinuousScalar == Position1 == Position{1}`
- [ ] `RoME.Point2 == Position2 == Position{2}`
- [ ] `RoME.Point3 == Position3 == Position{3}`

`RoME.Point2` and `RoME.Point3` already have name conflicts with JuliaGeometry.  `ContinuousScalar` seems out on its own.

Added benefit is users can then happily build higher dimensional stuff with `Position{4}`, etc.  It's then very likely that many use cases fall under `Manifolds.TranslationGroup(N)` and `IIF.Position{N}`.

**Comment by GearsAD** [comment]
:100: agreed! Can we do these as aliases and then deprecate the old naming over time?

**Reply by dehann** [reply]
Yes, full and healthy deprecation cycle should be easy to do here.

**Reply by dehann** [reply]
In Julia world we should be able to just use
```julia
@deprecate Point2(w...) Position2(w...)
```

**Comment by dehann** [comment]
and probably always keep the aliases:
```julia
const Position2 = Position{2}
```

**Comment by dehann** [comment]
Just adding technical detail:
```julia
getManifold(::Position{N}) where N = Manifolds.TranslationGroup(N)
```

**Comment by Affie** [comment]
> IIF.ContinuousScalar == Position1 == Position{1}

Scalar is a good generic name for me for applications that aren't spatial.

I like `Point{N}` more than `Position{N}`, but am fine with both.
